mw-server/internal/routers/metric.route_test.go

```diff
@@ -0,0 +1,58 @@
+package routers
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"mwserver/internal/auth"
+	"mwserver/internal/config"
+	"net/http"
+
+	openapiGeneral "mwserver/apiAutogenerated/general"
+	"mwserver/internal/openapi"
+	"testing"
+)
+
+func TestCreateMetric(t *testing.T) {
+	newConfig, err := config.LoadConfig("../../")
+	if err != nil {
+		t.Fatalf("Failed to load config: %v", err)
+	}
+	generalApi := openapi.MakeGeneralAPIClient(&newConfig)
+	_, err = generalApi.DevAPI.DevResetDbGet(context.Background()).Execute()
+	if err != nil {
+		t.Fatalf("Failed to reset db: %v", err)
+	}
+
+	currentUserID := "7cdb041b-4574-4f7b-a500-c53e74c72e94"
+	Description := "blablabla"
+	IsDone := false
+	MetricEstimation := 0
+	WayUuid := "550e8400-e29b-41d4-a716-446655440000"
+
+	t.Run("Should create metric successfully", func(t *testing.T) {
+		token, err := auth.GenerateJWT(currentUserID, newConfig.SecretSessionKey)
+		if err != nil {
+			t.Fatalf("Failed to generate JWT: %v", err)
+		}
+		request := openapiGeneral.SchemasCreateMetricPayload{
+			Description:    Description,
+			EstimationTime: int32(MetricEstimation),
+			IsDone:         IsDone,
+			WayUuid:        WayUuid,
+		}
+		ctx := context.WithValue(context.Background(), auth.ContextKeyAuthorization, "Bearer "+token)
+		metricResponse, response, err := generalApi.MetricAPI.CreateMetric(ctx).Request(request).Execute()
+		if err != nil {
+			t.Fatalf("Failed to create metric: %v", err)
+		}
+		expectedData := &openapiGeneral.SchemasMetricResponse{
+			Description:    Description,
+			EstimationTime: int32(MetricEstimation),
+			IsDone:         IsDone,
+		}
+		assert.Equal(t, http.StatusOK, response.StatusCode)
+		assert.Equal(t, expectedData.Description, metricResponse.Description)
+		assert.Equal(t, expectedData.EstimationTime, metricResponse.EstimationTime)
+		assert.Equal(t, expectedData.IsDone, metricResponse.IsDone)
+	})
+}
```

Comment:
lets check it with another appropriate endpoint also bc it is integrational test